doc/classes/CodeEdit.xml

```diff
@@ -146,6 +146,12 @@
 				Code regions are delimited using start and end tags (respectively [code]region[/code] and [code]endregion[/code] by default) preceded by one line comment delimiter. (eg. [code]#region[/code] and [code]#endregion[/code])
 			</description>
 		</method>
+		<method name="delete_lines">
+			<return type="void" />
+			<description>
+				Deletes all lines that are selected or have a caret on it.
```

Comment:
it -> them

The mistake is made multiple times; I won't spam review comments about it.